src/content/contributing/code-of-conduct/_index.en.md

```diff
@@ -4,3 +4,11 @@ date: 2020-02-19T21:43:46+01:00
 weight: 20
 ---
 
+## Submariner Community Code of Conduct
+
+Submariner follows the [CNCF Code of Conduct][1].
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the Submariner Committers.
```

Comment:
What happens if the instances involve committers? (Probably too big a discussion for a PR...)

Reply:
Yeah I thought about that and don't have a super clear option. Ideas:

1) Make sure we have enough committers that it's unlikely they are all involved.
2) Make an email alias like conduct@submariner.io that covers a larger group than just committers.
    * I don't like this option because it hides who will be contacted, so the person doing the reporting can't know who they are talking to, if maybe it's someone who was involved in the problem.

We also need some sort of contact page to link to this (if we keep it), but that's a different PR.

Reply:
Raised https://github.com/submariner-io/submariner-website/issues/64 to track improving reporting contact details.

Reply:
I'd think once one attains committer status they should be trusted not to engage in unacceptable behavior 😄

Reply:
@tpantelis that's been one of the biggest issues in the opensource industry, there are many examples (sadly)

Reply:
> I'd think once one attains committer status they should be trusted not to engage in unacceptable behavior smile

Unfortunately the situations which sparked the use of CoCs involved people in positions of trust and/or power. CoCs effectively have two main purposes: dealing with trolls, and dealing with people abusing their positions.